### PR TITLE
Add `CHANGELOG.md` to track release notes

### PR DESCRIPTION
**Pull request by agedd**
Previously, release notes were added onto the release tags. 

This PR creates a 'CHANGELOG` markdown file where we store notes on each release, starting from version 0.100.0. It is modeled after the current Braintree changelog.

Relevant JIRA Ticket: https://engineering.paypalcorp.com/jira/browse/DTNATIVEXO-1332

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Full EU support with 3DS support for contingency resolution
+  - For more information on how to enable 3DS support, see the README.md for instructions on installing the `CardinalMobile.xcframework`
+- Bug fixes and improved localizations
+- Improved accessibility support
+
+## 0.104.0 (2022-09-01)
+Previously validating an app while submitting to the store would result in a failure, this release fixes that issue.
+
+## 0.103.0 (2022-08-29)
+Previously, validating an app while submitting to the store would result in a failure relating to a dependency resource bundle and framework path. This release fixes that issue.
```
Comment (jonathajones) [inline CHANGELOG.md:34]: Could we format these the same way we format the other line items of the other releases?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- Improved accessibility support
+
+## 0.104.0 (2022-09-01)
+- Fixes an issue with app validation when submitting to the store.
+
+## 0.103.0 (2022-08-29)
+- Fixes an issue related to dependency resource bundle and framework path during app validation
+  while submitting to the store.
+
+## 0.102.0 (2022-09-22)
```
Comment (jonathajones) [inline CHANGELOG.md:37]:
```suggestion
## 0.102.0 (2022-08-22)
```

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
+</Workspace>
```
Comment (jonathajones) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:7]: do you know why this was added?

Comment (agedd) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:7]: hmm i'm not sure, i think it was automatically added when i created the changelog markdown file - but it doesn't seem relevant. maybe i can delete it?

Comment (jonathajones) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:7]: I'm in favor of removing if possible! 👍

**Review by jonathajones — APPROVED** [review]

**Pull request merged**